package.json

```diff
@@ -24,6 +24,7 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.18.2",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
```

Comment:
Some -proposal- plugins were renamed in stripes-webpack to -transform- because they became standard and will were renamed by babel.
https://github.com/folio-org/stripes-webpack/pull/114
Please rename the same plugins here and in babel.config.js